cocos/tween/actions/action-interval.ts

```diff
@@ -167,13 +167,13 @@ export class ActionInterval extends FiniteTimeAction {
         }
     }
 
-    startWithTarget (target: any) {
+    startWithTarget (target: any): void {
         Action.prototype.startWithTarget.call(this, target);
         this._elapsed = 0;
         this._firstTick = true;
     }
 
-    reverse () {
+    reverse (): this {
```

Comment:
Is `this` a valid type?

Reply:
it's valid type in TS, but it's better to mark it as the typeof this